docs/logs/settings.md

```diff
@@ -0,0 +1,84 @@
+title: Sematext Logs Settings
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
```

Comment:
I think I've seen this same description elsewhere. This is bad for SEO.

Reply:
Changed all descriptions to be unique